pkg/controller/postgresql.go

```diff
@@ -457,3 +463,74 @@ func (c *Controller) postgresqlDelete(obj interface{}) {
 
 	c.queueClusterEvent(pg, nil, spec.EventDelete)
 }
+
+/*
+  Ensures the pod service account and role bindings exists in a namespace before a PG cluster is created there so that a user does not have to deploy these credentials manually.
+  StatefulSets require the service account to create pods; Patroni requires relevant RBAC bindings to access endpoints.
+
+  The operator does not sync accounts/role bindings after creation.
+*/
+func (c *Controller) submitRBACCredentials(event spec.ClusterEvent) error {
+
+	namespace := event.NewSpec.GetNamespace()
+	if _, ok := c.namespacesWithDefinedRBAC.Load(namespace); ok {
```

Comment:
Now we use a single flag to indicate the namespace has service accounts and role binding defined, but in fact it may define the service account and fail on the role binding, so inside each of those functions we also need to handle "object already exist" (k8utils.ResourceAlreadyExists(err))  case as a non-error one.